osprey/eval_scopes.py

```diff
@@ -41,7 +41,8 @@ def estimator_in_module(mod):
             if ispkg:
                 result.update(import_all_estimators(mod))
             for kls in estimator_in_module(mod):
-                result[kls.__name__] = kls
+                if kls.__name__ not in result.keys():
```

Comment:
@mpharrigan: let me know if you think this is a satisfactory fix. Another condition could be to only allow a module to replace another if it's from `msmbuilder`.